.pipelines/soak-tests-template.yml

```diff
@@ -64,13 +64,10 @@ jobs:
           displayName: "Check cluster's health"
 
         - script: |
-            if [[ -n "${IMAGE_VERSION:-}" ]]; then
-              echo "Image version: ${IMAGE_VERSION}"
-              export MIC_VERSION="${IMAGE_VERSION}"
-              export NMI_VERSION="${IMAGE_VERSION}"
-              export IDENTITY_VALIDATOR_VERSION="${IMAGE_VERSION}"
-            fi
-            export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/k8s/aad-pod-identity}"
+            export MIC_VERSION="v1.6.3-rc.0"
```

Comment:
Why are we hardcoding the image versions and registry here? I know we setup the deployment and images in the cluster, but wondering if we should still keep it configurable like before instead of hardcoding it in here.

Reply:
It was a temporary change that I forgot to revert.